packages/paste-website/src/pages/components/form-pill-group/index.mdx

```diff
@@ -0,0 +1,434 @@
+  data={props.data.allPasteComponent.edges}
+  packageStatus={props.data.allAirtable.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview scope={{FormPill, FormPillGroup, useFormPillState, CalendarIcon}} noInline>
+  {defaultExample}
+</LivePreview>
+
+## Guidelines
+
+### About Form Pill Group
+
+<Paragraph>
+  {props.pageContext.frontmatter.description} They are used almost exclusively in multi-select editing situations.
+</Paragraph>
+
+A Form Pill Group can be used on it's own to represent selection across a number of fields, such as displaying currently applied filters when filtering data across a number of different fields. Alternatively, a Form Pill Group can be paired directly to a single input field, such as a Combobox, to represent multiple selection.
+
+#### Accessibility
+
+The Form Pill Group provides a number of accessibility features. The only requirement from you, is to provide the Pill Group a descriptive label via the `aria-label` React prop.
+
+##### Keyboard navigation
+
+The Form Pill Group is focusable, but only one pill is focusable at a time. This means the Pill Group is a single tab stop to a keyboard user. Once focused within the Pill Group the following keyboard interactions apply.
+
+- Left and Right arrow keys move focus within the group
+- If selectable, spacebar and enter will toggle pill selection
+- If a pill has a supplied action via `onClick`, enter key will trigger that action
+- If dismissable, a Pill can be removed by pressing the delete or backspace key
+
+<Box marginBottom="space70">
+  <AspectRatio ratio="16:9">
+    <iframe
+      src="https://www.loom.com/embed/1bea986ec6a24cd494950e2c12f10562"
+      frameborder="0"
+      webkitallowfullscreen
+      mozallowfullscreen
+      allowfullscreen
+      title="Silent demo on keyboard navigation"
+    ></iframe>
+  </AspectRatio>
+</Box>
+
+#### Display Pill vs Form Pill
+
+There are some very important semantic differences between a Display Pill and Form Pill group. Below is a table explaining the main differences to get a better understanding of when to use each type of Pill.
+
+<FormPillVsDisplayPillTable />
+
+## Examples
+
+### Basic Form Pill Group
+
+The basic Form Pill Group can be composed by adding `FormPill` components as children of `FormPillGroup`. These must be used with the `useFormPillState` hook, and the returned state should be spread onto each component.
+
+This will provide you with some internal state management and keyboard navigation.
+
+<LivePreview scope={{FormPill, FormPillGroup, useFormPillState, CalendarIcon}} noInline>
+  {defaultExample}
+</LivePreview>
+
+### Selectable Form Pill Group
+
+Outside of keyboard navigation, Form Pills are presentational. If you would like to make the Pill Group provide selection options, you can manage the state of those pills and seemlessly merge it with the state return from the `useFormPillState` hook.
```

Comment:
I think this paragraph could be clearer. I don't know what Form Pills being "presentational" means, and why that's different in comparison to keyboard nav. Also it would be useful to provide a use case example for making the Form Pill Group "provide selection options", since I'm having a hard time picturing what's meant by that as well.

Reply:
I think I made it better?